tools/tcpconnlat.py

```diff
@@ -40,6 +40,8 @@ def positive_float(val):
     ./tcpconnlat -t        # include timestamps
     ./tcpconnlat -p 181    # only trace PID 181
     ./tcpconnlat -L        # include LPORT while printing outputs
+    ./tcpconnlat -4        # trace IPv4 family only
+    ./tcpconnlat -6        # trace IPv6 family ony
```

Comment:
family only

Reply:
Done. Fixed typo and added options to man pages.